secrethub_wrapper.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+/*
+typedef long long time;
+typedef char* uuid;
+
+struct Secret {
+	uuid SecretID;
+	uuid DirID;
+	uuid RepoID;
+	char* Name;
+	char* BlindName;
+	int VersionCount;
+	int LatestVersion;
+	char* Status;
+	time CreatedAt;
+};
+
+struct SecretVersion {
+	uuid SecretVersionID;
+	struct Secret Secret;
+	int Version;
+	char* Data;
+	time CreatedAt;
+	char* Status;
+};
+
+struct Client {
+	int ID;
+};
+#include <stdbool.h>
+#include <stdlib.h>
+*/
+import "C"
+
+import (
+	"encoding/json"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"os"
+	"strings"
+	"unsafe"
+)
+
+var clients = make(map[int]secrethub.ClientInterface)
+var nextClientID = 1;
```

Comment:
This is an int so there is technically a limit of `4294966654` clients you can have during the runtime of the program (if you do not delete the first one). I assume this is not a big issue for now, but let me know if it should be fixed.